### PR TITLE
Bug 1123164 - [Settings]There is no SIM settings icons when you insert SIM2 only in Settings -> Cellular & Data.

### PR DESCRIPTION
**Pull request by mpizza**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen, Gary (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0cba3a3bf688)

File: apps/settings/js/main.js [hunk 1]
```diff
@@ -35,20 +36,35 @@ require(['config/require'], function() {
...
+              // If the device support dsds,
+              // then navigate to 'call-iccs' panel
+              pendingTargetPanel = 'call-iccs';
+            }
+            SettingsService.navigate(pendingTargetPanel);
+            break;
+          default:
+            if (pendingTargetPanel) {
+              SettingsService.navigate(pendingTargetPanel);
             }
```
Comment (crh0716) [inline apps/settings/js/main.js]: nit: Let's still add a break here.

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen, Gary (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=835e9486319b)

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a4efa2a06f67)

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=61585286d654)

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=eae8a25c9e72)

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ff273920c310)

File: apps/settings/js/main.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ require(['config/require'], function() {
     // The following are the scripts used by many other scripts. We load them
     // at once here. These should be move to the dependency of each panel in the
     // future.
+    var DsdsSettings = require('dsds_settings');
```
Comment (crh0716) [inline apps/settings/js/main.js]: nit: Please move this to the line under `var Settings = require('settings');`.

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f9585099d33d)

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c16da1f625ef)

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7f09f28ab68f)

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b2f72635656a)

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a9bf40c3b57a)

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=75b223fbd5c9)

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dbad97177f49)

**Comment by try-server-hook** [comment]
[mpizza GaryChen, PYChen (mpizza)](https://github.com/mpizza) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0552403b3c26)

**Pull request merged**